### PR TITLE
docs: template for component docs

### PR DESCRIPTION
**Pull request by sebald**
Ref #1832

### CODE REVIEW

**Comment by sebald** [comment]
We should create a `<PropsTable>` and wrapper (that includes the grid/stack stuff` for the Dos and Don'ts

File: docs/templates/component.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+figma: <link to figma> (optional)
+---
+
+<!-- longer description about capabilities and usage of the component -->
+
+```tsx onlyCode
+import {} from '@marigold/';
+```
+
+## Examples
```
Comment (ti10le) [inline docs/templates/component.mdx:13]: Examples is like showing different props in action or dont show every prop separate in an example?

File: docs/templates/component.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+<!-- show some cases and scenarios in which the component is used -->
+
+### Use case
```
Comment (ti10le) [inline docs/templates/component.mdx:17]: Use case should be one or two cases or what do you think about it?

File: docs/templates/component.mdx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+<!-- describe the use case -->
+
+```tsx
+<div>Hello there!</div>
+```
+
+## Props
```
Comment (ti10le) [inline docs/templates/component.mdx:25]: I would move this under the import

**Review by ti10le — DISMISSED** [review]

File: docs/templates/component.mdx (hunk 1 above)
Comment (sebald) [reply docs/templates/component.mdx:13]: yes, see description in comment

File: docs/templates/component.mdx (hunk 2 above)
Comment (sebald) [reply docs/templates/component.mdx:17]: This is only meant as a placeholder, it's "examples" -> "primary button" for example and the latter is the "use case"

**Review by ti10le — APPROVED** [review]

**Pull request merged**